Mathlib/Data/Int/Cast/Basic.lean

```diff
@@ -64,7 +64,7 @@ theorem cast_zero : ((0 : ℤ) : R) = 0 :=
 @[simp high, nolint simpNF, norm_cast] -- this lemma competes with `Int.ofNat_eq_cast` to come later
 theorem cast_ofNat (n : ℕ) : ((n : ℤ) : R) = n :=
   AddGroupWithOne.intCast_ofNat _
-#align int.cast_coe_nat Int.cast_ofNatₓ
+#align int.cast_nat_cast Int.cast_ofNatₓ
```

Comment:
Why did the lean3 half of the align change?

Reply:
Ah shoot, I was going through the changed `#align`s and reverting the changes, but I misreverted this one